.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ orbs:
 jobs:
   build:
     docker:
-      - image: 'citusdata/extbuilder:werror'
```

Comment:
is `latest` now `werror`?

Reply:
Yeah, I heard that `werror` tag wasnt necessary anymore anyways since it was merged